packages/broadcast/component.js

```diff
@@ -48,30 +53,37 @@ export class FabricBroadcast extends LitElement {
   }
 
   async _del(id) {
-    const el = this.renderRoot.querySelector(`#${id}`);
+    const el = this.renderRoot.querySelector(`#broadcast-${id}`);
     await el.collapse();
+    this.hiddenMessageIds = [...this.hiddenMessageIds, id];
```

Comment:
I could see the possibility for an edge case where the same id was clicked several times which would result in this array containing the same id several times. Unlikely to cause much trouble but it could be avoided by using a JS Set to ensure unique values.

Reply:
I think a quick conditional perhaps would suffice also? Although, a Set is definitely nicer as it avoids duplicates by default.

Reply:
GP! Used `Set` in [9da5ddb](https://github.com/fabric-ds/elements/pull/61/commits/9da5ddbf339fa80aa2226fa0ec4834263b006ee3)